### DISCUSSION TITLE
About accept() call

### DISCUSSION THREAD

**Discussion by lgm9** [ANLAB-KAIST/KENSv3#72] [Project Q&A 2. Transport Layer (TCP)] [answered]
![캡처1](https://user-images.githubusercontent.com/56072520/162607912-97801928-3f39-439c-94fe-06b01b133266.PNG)
![캡처2](https://user-images.githubusercontent.com/56072520/162607908-a04d4e13-4281-499f-abe5-d679fb49f1ba.PNG)
![캡처3](https://user-images.githubusercontent.com/56072520/162607910-30d9a2a4-b0d7-4e00-a984-330f940342fc.PNG)

I made some printf debug code at
1. In testclose.cpp, right before calling accept()
2. In TCPAssignment.cpp, start of the ACCEPT system call.
However, in the actual run, the call of accept() is successful, but my code at the system call ACCEPT does not run. Is it normal?

**Comment by Pusnow** [comment, marked as answer]
> However, in the actual run, the call of accept() is successful, but my code at the system call ACCEPT does not run. Is it normal?

The test case (`TestClose_Connect_CloseFirst`) check how you implement `connect()`.
Therefore, it uses the `accept()` implementation in the solution binary.
